pages/new/index.html

```diff
@@ -29,13 +29,28 @@
         },
         computed: {
           compiledMarkdown() {
-            setTimeout(() => {
```

Comment:
Why remove the setTimout... iirc after Vue rerenders it should be after 0 milliseconds.

Reply:
@TheColaber Using requestAnimationFrame is better practice and more optimized; before the scratchblocks used to flash but now they don't.

Reply:
Seems to be blinking more for me...

https://user-images.githubusercontent.com/72760579/112247374-8f8f9980-8c2a-11eb-8285-33ea54c72c83.mp4

Reply:
huh, but the scratchblocks don't blink at all... interesting...

Reply:
> huh, but the scratchblocks don't blink at all... interesting...

because scratchblocks aren't really code blocks

Reply:
@CubeyTheCube Seems like the function itself... `Rainbow.color();` is causing the blinking... do you know how to disable it and/or pick a different one?